### PR TITLE
Add AutoRemesher v1.0.0-beta.3

### PR DESCRIPTION
**Pull request by ethanproia**
Automatic quad geometry retopology/remesh.
`Livecheck` stanza may need fixing

### CODE REVIEW

File: Casks/autoremesher.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask "autoremesher" do
+  version "1.0.0-beta.3"
+  sha256 :no_check
```
Comment (p-linnane) [inline Casks/autoremesher.rb:3]:
```suggestion
  sha256 "be8450f69d8c52e1b7cd9d3e7ecacfe6bd64fa7420c6fa3944949509338a85a7"
```

File: Casks/autoremesher.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  url "https://github.com/huxingyi/autoremesher/releases/download/#{version}/autoremesher-#{version}.dmg"
+  name "AutoRemesher"
+  desc "Your gateway to Megascans and Metahumans"
+  homepage "https://github.com/huxingyi/autoremesher/"
+
+  livecheck do
+    url "https://github.com/huxingyi/autoremesher/releases/"
+    strategy :header_match
```
Comment (p-linnane) [inline Casks/autoremesher.rb:12]:
```suggestion
    url :url
    regex(/^v?(\d+(?:\.\d+)+(?:-beta\.?\d*)?)$/i)
```

**Comment by p-linnane** [comment]
CI is failing because the application is not `codesigned`. Please contact the upstream developer and ask them to `codesign` their application. Otherwise this will prevent the application from running on Apple Silicon machines without disabling Gatekeeper (which is not ideal).

**Comment by miccal** [comment]
Thank you for your submission @ethanproia, but we cannot accept it at this time because it does not pass our signature verification check:

> macOS on ARM requires applications to be signed. Please contact the upstream developer to let them know they should sign their app.

Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if the developers sign their app in the future we would then be happy to consider adding this to `homebrew-cask`.

**Pull request closed without merging**